src/osd/OSDCap.h

```diff
@@ -222,14 +222,15 @@ ostream& operator<<(ostream& out, const OSDCapGrant& g);
 
 
 struct OSDCap {
+  CephContext* cct = nullptr;
```

Comment:
OK, so we slightly inflate (by one pointer) the size of `OSDCap` and `MonCap`. I think it's worth doing that but I assumed neither of types has tons of instances.